pkg/timeline/adaptor/repository/dummy.ts

```diff
@@ -124,6 +124,15 @@ export class InMemoryListRepository implements ListRepository {
     return Result.ok(list);
   }
 
+  async fetchListsByMemberAccountID(
+    accountID: AccountID,
+  ): Promise<Result.Result<Error, List[]>> {
+    const lists = [...this.listData.values()].filter((list) =>
+      list.getMemberIds().includes(accountID),
+    );
+    return Result.ok(lists.map((list) => list));
```

Comment:
```suggestion
    return Result.ok(lists);
```
この `map` は効果が無いと思います.